build/android/Makefile

```diff
@@ -198,7 +192,11 @@ $(PATHCFGFILE) :
 		echo "$$SDKFLDR is not a valid folder";                                \
 		exit 1;                                                                \
 	fi;                                                                        \
-	echo "SDKFOLDER = $$SDKFLDR" >> ${PATHCFGFILE};
+	echo "SDKFOLDER = $$SDKFLDR" >> ${PATHCFGFILE};                            \
+
+local.properties: $(PATHCFGFILE)
+	@echo "sdk.dir = $$SDKFLDR" >> local.properties;
```

Comment:
I'm not sure but if you do make -j 2 is it possible that this line and the line below get executed in parrallel and cause lots of trouble inside the file? What about `echo [..] >> local.properties && \ \n echo [..] >> local.properties;` ?